src/libs/GetStyledTextArray.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Render a suggestion menu item component.
+ * @param {String} name
+ * @param {String} prefix
+ * @returns {Array}
+ */
+const getStyledTextArray = (name, prefix) => {
+    const texts = [];
+    const prefixLocation = name.search(prefix);
+
+    if (prefixLocation === 0 && prefix.length === name.length) {
+        texts.push({text: prefix, isColored: true});
```

Comment:
text should be `name` instead of `prefix`.
Later, this caused minor regression - https://github.com/Expensify/App/issues/32749